examples/asr/speech_to_text.py

```diff
@@ -12,89 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# TODO: This is WIP and needs a lot of polishing
-# python speech_to_text.py \
-#         --asr_model "bad_quartznet15x5.yaml" \
-#         --train_dataset "./an4/train_manifest.json" \
-#         --eval_dataset "./an4/test_manifest.json" \
-#         --gpus 4 \
-#         --distributed_backend "ddp" \
-#         --max_epochs 1 \
-#         --fast_dev_run \
-#         --lr 0.001 \
-
-from argparse import ArgumentParser
-
+import hydra
 import pytorch_lightning as pl
-from ruamel.yaml import YAML
 
 from nemo.collections.asr.models import EncDecCTCModel
-from nemo.core.optim.lr_scheduler import CosineAnnealing
-from nemo.utils.arguments import add_asr_args, add_optimizer_args, add_scheduler_args
 
 
-def main(args):
-
-    yaml = YAML(typ="safe")
-    with open(args.asr_model) as f:
-        model_config = yaml.load(f)
+@hydra.main(config_path="conf", config_name="config")
+def main(cfg):
+    print(cfg.pretty())
```

Comment:
Remove or change the print() statement

Reply:
I'd like to keep some form of this for now. Can you elaborate on what you would like to be printed / not printed?

Reply:
I am for leaving this up for now.

Bit generally you can always show parameters by:
```
(nemo-env) tkornuta@tkornuta-desktop:~/workspace/nemo/examples/cv$ python mnist_lenet5_image_classification_pure_lightning.py -h
mnist_lenet5_image_classification_pure_lightning is powered by Hydra.

== Configuration groups ==
Compose your configuration from those groups (group=option)



== Config ==
Override anything in the config (foo.bar=value)

name: Training of a LeNet-5 Model using a pure PyTorchLightning approach - using DDP
  on 2 GPUs.
trainer:
  gradient_clip_val: 0.0
  process_position: 0
  num_nodes: 1
  num_processes: 1
  gpus: 2
  auto_select_gpus: false
  log_gpu_memory: null
```

Reply:
Or show configuration by: 
```
(nemo-env) tkornuta@tkornuta-desktop:~/workspace/nemo/examples/cv$ python mnist_lenet5_image_classification_pure_lightning.py --cfg job
name: Training of a LeNet-5 Model using a pure PyTorchLightning approach - using DDP
  on 2 GPUs.
trainer:
  gradient_clip_val: 0.0
  process_position: 0
  num_nodes: 1
  num_processes: 1
  gpus: 2
  auto_select_gpus: false
  log_gpu_memory: null
  progress_bar_refresh_rate: 1
  check_val_every_n_epoch: 1
  fast_dev_run: false
  max_epochs: 1000
  min_epochs: 1
  distributed_backend: ddp
```

Reply:
As in do not use `print()`, use logging if you want

Reply:
Changed print to logging.